IDEAS/Buildings/Components/Zone.mo

```diff
@@ -6,46 +6,92 @@ model Zone "thermal building zone"
   extends IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations(
     energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
     redeclare replaceable package Medium = IDEAS.Media.Air,
-    final mSenFac = corrCV);
+    final mSenFac = 1);
```

Comment:
Yup, I'll remove the extends clause from `IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations` as suggested.